root/edit/details/AddRemoveAlias.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+/* eslint-disable no-negated-condition */
+import React from 'react';
+
+import bracketed from '../../static/scripts/common/utility/bracketed';
+import formatEntityTypeName from '../../static/scripts/common/utility/formatEntityTypeName';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import isDateEmpty from '../../static/scripts/common/utility/isDateEmpty';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import isolateText from '../../static/scripts/common/utility/isolateText';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AliasEntityTypeT = $ElementType<AliasEntityTypeT, 'entityType'>;
+
+type DisplayDataT = {
+  alias: string,
+  begin_date: PartialDateT,
+  end_date: PartialDateT,
+  ended: boolean,
+  entity_type: AliasEntityTypeT,
+  locale: string,
+  primary_for_locale: boolean,
+  sort_name: string,
+  type: AliasT,
+};
+
+type Props = {
+  ...EditT,
+  display_data: DisplayDataT,
+};
+
+const AddRemoveAlias = ({edit}: {edit: Props}) => {
+  const display = edit.display_data;
+  const entityType = display.entity_type;
+  const entity = display[entityType];
+  return (
+    <table className={`details ${edit.edit_kind}-${entityType}-alias`}>
+      <tr>
+        <th>{addColon(formatEntityTypeName(entityType))}</th>
+        <td>
+          <DescriptiveLink entity={entity} />
+          {' '}
+          {(entity && entity.gid) ? bracketed(<EntityLink content={l('view all aliases')} entity={entity} subPath="aliases" />) : null}
+        </td>
+      </tr>
+      <tr>
+        <th>{addColon(l('Alias'))}</th>
+        <td>{isolateText(display.alias)}</td>
+      </tr>
+      {(display.sort_name !== '') ? (
+        <tr>
+          <th>{addColon(l('Sort name'))}</th>
+          <td>{display.sort_name}</td>
+        </tr>
+      ) : null}
+      {(display.locale) ? (
+        <>
+          <tr>
+            <th>{addColon(l('Locale'))}</th>
+            <td>{display.locale}</td>
+          </tr>
+          <tr>
+            <th>{addColon(l('Primary for locale'))}</th>
+            <td>{yesNo(display.primary_for_locale)}</td>
+          </tr>
+        </>
+      ) : null}
+      {(display.type) ? (
+        <tr>
+          <th>{addColon(l('Type'))}</th>
+          <td>{display.type.name}</td>
```

Comment:
```suggestion
          <td>{lp_attributes(display.type.name, 'alias_type')}</td>
```
to translate the type name.

Reply:
why is this done ?

Reply:
`display.type.name` is the type name in English. `lp_attributes(display.type.name, 'alias_type')` passes it to gettext (which we use to translate strings). So if someone is browsing MusicBrainz in another language, it'll display the type name in their language. :)